packages/flutter/lib/src/material/text_field.dart

```diff
@@ -369,6 +370,13 @@ class TextField extends StatefulWidget {
   /// {@end-tool}
   final TextMagnifierConfiguration? magnifierConfiguration;
 
+  /// {@template flutter.widgets.editableText.groupId}
```

Comment:
Could you use a documentation macro so that the docs are not duplicated?

Reply:
Of course, thank you for your suggestion